packages/bridge-schema/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@nuxt/bridge-schema",
+  "version": "3.0.0",
+  "repository": "nuxt/bridge",
+  "license": "MIT",
+  "type": "module",
+  "main": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "schema"
+  ],
+  "scripts": {
+    "build": "unbuild",
+    "prepack": "unbuild"
+  },
+  "devDependencies": {
+    "@types/lodash.template": "^4",
+    "@types/semver": "^7",
+    "@vitejs/plugin-vue": "^3.1.2",
+    "@vueuse/head": "~1.0.0-rc.9",
+    "nitropack": "^0.6.0",
```

Comment:
I think we can update nitropack and other deps here as well? (in other package.json it is updated)

Reply:
thanks!